requirements.txt

```diff
@@ -34,8 +34,8 @@ Pillow==5.1.0
 django-storages==1.6.6
 boto3==1.7.50
 
-# dbca for SSO login
-git+https://github.com/dbca-wa/dpaw-utils.git@0.4.1
+# dbca-utils for SSO login
+https://static.dbca.wa.gov.au/static/py/dpaw-utils/dist/dpaw-utils-0.3a15.tar.gz
```

Comment:
Hmm, I can't remember why I did that. I had to use a fix in 0.4.1 that was not in the static. Not sure. I will investigate.

Reply:
That's right. I was testing Biosys with Django 2.0 and that was a blocker. Could you put the the 0.4.1 version in your static cdn?